pos_pricelist/static/src/js/models.js

```diff
@@ -15,29 +15,62 @@
  *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  ******************************************************************************/
-function pos_pricelist_models(instance, module) {
+odoo.define('pos_pricelist.models', function (require) {
+	"use strict";
 
-    var _t = instance.web._t;
-    var round_pr = instance.web.round_precision;
-    var round_di = instance.web.round_decimals;
+	var models = require('point_of_sale.models');
+	var screens = require('point_of_sale.screens');
+	var PosDB = require('pos_pricelist.DB');
+	var core = require('web.core');
+	var utils = require('web.utils');
 
-    /**
+    var round_pr = utils.round_precision;
+    var round_di = utils.round_decimals;
+
+	PosDB = PosDB.extend({
```

Comment:
I don't understand this changes. you moved PosDB definition from db.js to models.js ?

Reply:
You should use include instead of extend.